scripts/wheeler_test.sh

```diff
@@ -9,7 +9,7 @@ CONTAINER_NAME="das-node-wheeler-test"
 docker run \
   --rm \
   --name=$CONTAINER_NAME \
-  --mount type=volume,source=bazel_cache,target=/root/.cache/bazel \
+  --volume ./docker/volumes/bazel_cache:/root/.cache/bazel \
```

Comment:
Is there any reason for this change? Just want to understand it, not particularly against it.

Reply:
Yes, when a Docker volume is created, its mount point is located inside /var. Since I'm using actions/cache to share the volume globally in the pipeline, this setup was causing permission issues. That's why I switched to a bind mount instead, as it lets me set the mount point to a directory within the working directory, avoiding these kinds of issues.